Implemented gcal-default-setting and fixed some bugs with gcal-integratio

StingerAJ opened the pull request:
Implemented gcal-default-setting and fixed some bugs with gcal-integration (timezone-bug, estimated time, adding duetime, changing task-details). This setting is not regarded for sync, as the webservices usually offer their own gcal-integration like gtasks, producteev.

Please wait for Malika checking it thoroughly before you integrate this pull-request.

StingerAJ committed:
Implemented gcal-default-setting and fixed some bugs with gcal-integration (timezone-bug, estimated time, adding duetime, changing task-details). This setting is not regarded for sync, as the webservices usually offer their own gcal-integration like gtasks, producteev.

jonparis commented:
Arne and Malika are you able to test this?
Send me an update.

On Thursday, September 29, 2011, StingerAJ <
reply@reply.github.com>
wrote:

> Implemented gcal-default-setting and fixed some bugs with gcal-integration
> (timezone-bug, estimated time, adding duetime, changing task-details). This
> setting is not regarded for sync, as the webservices usually offer their own
> gcal-integration like gtasks, producteev.
> 
> Please wait for Malika checking it thoroughly before you integrate this
> pull-request.
> 
> You can merge this Pull Request by running:
> 
>  git pull https://github.com/StingerAJ/astrid110928_aj_gcal_default_setting
> 
> Or you can view, comment on it, or merge it online at:
> 
>  https://github.com/todoroo/astrid/pull/90
> 
> -- Commit Summary --
> - Implemented gcal-default-setting and fixed some bugs with
>   gcal-integration (timezone-bug, estimated time, adding duetime, changing
>   task-details). This setting is not regarded for sync, as the webservices
>   usually offer their own gcal-integration like gtasks, producteev.
> 
> -- File Changes --
> 
> M astrid/plugin-src/com/todoroo/astrid/core/DefaultsPreferences.java (20)
> M astrid/plugin-src/com/todoroo/astrid/gcal/Calendars.java (80)
> M astrid/plugin-src/com/todoroo/astrid/gcal/GCalControlSet.java (55)
> M astrid/plugin-src/com/todoroo/astrid/gcal/GCalHelper.java (90)
> M
> astrid/plugin-src/com/todoroo/astrid/repeats/RepeatTaskCompleteListener.java
> (19)
> M astrid/res/values-de/strings.xml (2)
> M astrid/res/values/keys.xml (5)
> M astrid/res/values/strings-core.xml (61)
> A astrid/res/values/strings-defaults.xml (70)
> M astrid/res/xml/preferences_defaults.xml (5)
> M astrid/src/com/todoroo/astrid/activity/TaskEditActivity.java (8)
> M astrid/src/com/todoroo/astrid/activity/TaskListActivity.java (9)
> 
> -- Patch Links --
> 
>  https://github.com/todoroo/astrid/pull/90.patch
>  https://github.com/todoroo/astrid/pull/90.diff
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/todoroo/astrid/pull/90

## 

Jon Paris | Co-Founder | Astrid | astrid.com | (415) 335-6647
We're hiring! astrid.com/jobs

StingerAJ commented:
I already tested it, Malika, please give it a spin...

Arne
Am 30.09.2011 09:55 schrieb "Jon" <
reply@reply.github.com>:

> Arne and Malika are you able to test this?
> Send me an update.
> 
> On Thursday, September 29, 2011, StingerAJ <
> reply@reply.github.com>
> wrote:
> 
> > Implemented gcal-default-setting and fixed some bugs with
> > gcal-integration
> > (timezone-bug, estimated time, adding duetime, changing task-details).
> > This
> > setting is not regarded for sync, as the webservices usually offer their
> > own
> > gcal-integration like gtasks, producteev.
> > 
> > Please wait for Malika checking it thoroughly before you integrate this
> > pull-request.
> > 
> > You can merge this Pull Request by running:
> > 
> > git pull
> > https://github.com/StingerAJ/astrid110928_aj_gcal_default_setting
> > 
> > Or you can view, comment on it, or merge it online at:
> > 
> > https://github.com/todoroo/astrid/pull/90
> > 
> > -- Commit Summary --
> > - Implemented gcal-default-setting and fixed some bugs with
> >   gcal-integration (timezone-bug, estimated time, adding duetime, changing
> >   task-details). This setting is not regarded for sync, as the webservices
> >   usually offer their own gcal-integration like gtasks, producteev.
> > 
> > -- File Changes --
> > 
> > M astrid/plugin-src/com/todoroo/astrid/core/DefaultsPreferences.java (20)
> > M astrid/plugin-src/com/todoroo/astrid/gcal/Calendars.java (80)
> > M astrid/plugin-src/com/todoroo/astrid/gcal/GCalControlSet.java (55)
> > M astrid/plugin-src/com/todoroo/astrid/gcal/GCalHelper.java (90)
> > M
> 
> astrid/plugin-src/com/todoroo/astrid/repeats/RepeatTaskCompleteListener.java
> (19)
> 
> > M astrid/res/values-de/strings.xml (2)
> > M astrid/res/values/keys.xml (5)
> > M astrid/res/values/strings-core.xml (61)
> > A astrid/res/values/strings-defaults.xml (70)
> > M astrid/res/xml/preferences_defaults.xml (5)
> > M astrid/src/com/todoroo/astrid/activity/TaskEditActivity.java (8)
> > M astrid/src/com/todoroo/astrid/activity/TaskListActivity.java (9)
> > 
> > -- Patch Links --
> > 
> > https://github.com/todoroo/astrid/pull/90.patch
> > https://github.com/todoroo/astrid/pull/90.diff
> > 
> > ## 
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/todoroo/astrid/pull/90
> 
> ## 
> 
> Jon Paris | Co-Founder | Astrid | astrid.com | (415) 335-6647
> We're hiring! astrid.com/jobs
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/todoroo/astrid/pull/90#issuecomment-2247062
